src/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -16,8 +16,9 @@ public class SendPacketsAsync
         private readonly ITestOutputHelper _log;
 
         private IPAddress _serverAddress = IPAddress.IPv6Loopback;
-        // In the current directory
-        private const string TestFileName = "NCLTest.Socket.SendPacketsAsync.testpayload";
+        // Accessible directories for UWP app:
+        // C:\Users\<UserName>\AppData\Local\Packages\<ApplicationPackageName>\
+        private string TestFileName = Environment.GetEnvironmentVariable("LocalAppData") + @"\NCLTest.Socket.SendPacketsAsync.testpayload";
```

Comment:
>How will this change affect these tests when they run on *Nix with non-UWP (i.e. .NET Core)? And did you re-test the non-UWP Windows tests as well?

No I haven't, I will re-test the non-UWP Windows tests, and wait for the CI result and make changes accordingly for *Nix with non-UWP tests